example/hidden.js

```diff
@@ -20,6 +22,6 @@ app.model(User);
 var apiPath = '/api';
 explorer(app, { basePath: apiPath });
 app.use(apiPath, loopback.rest());
-console.log('Explorer mounted at localhost:' + port + '/explorer');
+g.log('{{Explorer}} mounted at {{localhost:%s/explorer}}', port);
```

Comment:
@richardpringle It is only for words with ALL caps. ie. `API` that don't get translated or any words you put within the double brace. In your case, both `Explorer` and the url will not be translated.